geopyspark/tests/base_test_class.py

```diff
@@ -4,4 +4,4 @@
 
 
 class BaseTestClass(unittest.TestCase):
-    geopysc = GeoPyContext(master="local[*]", appName="test")
+    geopysc = GeoPyContext(master="local[1]", appName="test")
```

Comment:
Right, I thought about that, but it also slightly speeds up the tests locally.

Also, the tests are probably going to be run on Travis more consistently than locally, so it is arguable whether different behavior on Travis versus local are useful (that is one questions that I want to open up for discussion).